app/channel.go

```diff
@@ -91,17 +91,10 @@ func (a *App) JoinDefaultChannels(c request.CTX, teamID string, user *model.User
 		}
 	}
 
-	var err *model.AppError
```

Comment:
Wondering how this didn't caught by the compiler or linter as the err is not used anywhere 🤔

Reply:
It war read in line 100 and written in the lines 101 and 103, that's all it needs to be considered a useful member of ~society~ code by the compiler.